webcompat/form.py

```diff
@@ -329,6 +329,8 @@ def build_details(details):
 
 def get_radio_button_label(field_value, label_list):
     """Return human-readable label for problem choices form value."""
+    if len(label_list[0]) == 3:
```

Comment:
Here are we checking the length of the first item of `label_list` and not `label_list` itself? 

Can we update `test_radio_button_label` to test a label_list with 2 and 3 members? Right now it looks like it only tests that one scenario.

Reply:
oh good catch, I have forgotten. Thanks.

Reply:
done.